backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelper.java

```diff
@@ -226,6 +234,35 @@ public int countTakenCores(VDS host) {
         return numOfTakenCores;
     }
 
+    public int countUnavailableCpus(VDS host,
```

Comment:
maybe 'countUnavailableCores'? and how about adding a comment here? I saw you described it in another comment but it's missing here

Reply:
I prefer to call it CPU when we cannot say if we count threads or cores.

Reply:
I look at it differently - it's always about 'cores', that setting of treating threads as cores just determines if those are real, physical, cores or physical threads

Reply:
Or is it all about the CPUs? :-)

In any case, we should use the same term in all variables and methods in `CpuPinningUnit`. When I look at the current implementation of `CpuPinningUnit` I think it is pretty clear what we're counting and I would prefer not to rename everything.

Reply:
you'll maintain this code so if it makes more sense to you this way then ok but really, a comment would be better here because it's not so clear what it means when you're not in the right mindset :)

Reply:
ok, I thought it is commented in `CPUPolicyUnit` but you're right, the method could be accessed from elsewhere and it should be clear what it does